app/src/lib/stores/helpers/show-notification.ts

```diff
@@ -0,0 +1,20 @@
+import { remote } from 'electron'
+
+/**
+ * Shows a notification with a title, a body, and a function to handle when the
+ * user clicks on the notification.
+ */
+export function showNotification(
+  title: string,
+  body: string,
+  onClick: () => void
+) {
+  const notification = new Notification(title, {
+    body,
+  })
+
+  notification.onclick = () => {
+    remote.getCurrentWindow().focus()
```

Comment:
This should be replaced with something better other than `remote`. Probably worth doing the change after #13702 is merged.

Reply:
I will catch this with the remote 16 upgrade refactors (probably will already be making a proxy method for it)